<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_file">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>50</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Выбрать файл</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_file">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>90</y>
      <width>361</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="GlView" name="view_window" native="true">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>9</y>
      <width>561</width>
      <height>631</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>130</y>
      <width>431</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_info">
     <attribute name="title">
      <string>Информация</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>391</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Количество вершин</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Имя файла</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Количество ребер</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="file_name">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="count_vertices">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="count_edges">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_changes">
     <attribute name="title">
      <string>Изменение положения</string>
     </attribute>
     <widget class="QLabel" name="error_xyz_scale">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>300</y>
        <width>261</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>321</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QLineEdit" name="x_move">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_y">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_z">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_x">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="y_move">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLineEdit" name="z_move">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_move">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Переместить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_scale">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>260</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Масштабировать</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="scale">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>260</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rotate">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Повернуть</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>321</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QLineEdit" name="x_rotate">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_y_2">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_z_2">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_x_2">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="y_rotate">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLineEdit" name="z_rotate">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_settings">
     <attribute name="title">
      <string>Настройки</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>121</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Проекция</string>
      </property>
     </widget>
     <widget class="QComboBox" name="projection_type">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>42</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Центральная</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Параллельная</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>121</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ребра</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>121</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Толщина</string>
      </property>
     </widget>
     <widget class="QComboBox" name="line_type">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>132</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Сплошная</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Пунктирная</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>121</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Тип линии</string>
      </property>
     </widget>
     <widget class="QComboBox" name="vertices_type">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>260</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Отсутствует</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Круг</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Квадрат</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>258</y>
        <width>171</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Способ отображения</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>121</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Вершины</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>220</y>
        <width>121</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Размер</string>
      </property>
     </widget>
     <widget class="QComboBox" name="width_edge">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>92</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="size_vertices">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>222</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_color_edges">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>175</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Выбрать цвет</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_color_vertices">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>303</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Выбрать цвет</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>121</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Фон</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_color_back">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>365</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Выбрать цвет</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_save_image">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>610</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Сохранить изображение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_make_screencast">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>610</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Записать скринкаст</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1072</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GlView</class>
   <extends>QWidget</extends>
   <header>../../../../src/view/glview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
